pkg/asset/releaseimage/default.go

```diff
@@ -20,7 +20,7 @@ import (
 
 var (
 	// defaultReleaseImageOriginal is the value served when defaultReleaseImagePadded is unmodified.
-	defaultReleaseImageOriginal = "registry.ci.openshift.org/origin/release:4.12"
+	defaultReleaseImageOriginal = "registry.ci.openshift.org/origin/release:4.13"
```

Comment:
There is a ticket for this: https://issues.redhat.com/browse/CORS-2235
We can at least reference it in the patch headline.